src/ol_openedx_course_export/utils.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+
+from ol_openedx_course_export.constants import AWS_S3_DEFAULT_URL_PREFIX
+
+
+def is_bucket_configuration_valid():
+    """
+    For course export to work properly we need all the AWS settings configured properly
+    """
+    return None not in [
+        settings.COURSE_IMPORT_EXPORT_BUCKET,
+        settings.AWS_ACCESS_KEY_ID,
+        settings.AWS_SECRET_ACCESS_KEY,
+    ]
+
+
+def get_clean_file_name(tarball_name):
+    """
+    Args:
+        tarball_name: (str) name of the generated course tarball/OLX file
+        e.g. '/tmp/course.l8dtvuog.tar.gz'
+    Returns:
+        str: Returns the file name without prefix
+    """
+    # Since the generated tarball has associated prefix path e.g. "/tmp/" so we trim the path and pick the name
+    return tarball_name.rsplit("/")[-1]
```

Comment:
Would it be possible to build a path name that incorporates the course ID? If it's more than a ~15 minute effort then we'll punt on it for now.

Reply:
@blarghmatey I am a little confused on this but let me try to see the possibility of what you are thinking to make sure I'm in the right direction?

1) Are you referring to `/tmp/course.l8dtvuog.tar.gz` to incorporate course ID e.g. you want the tarball name to be something like `/tmp/course-v1:edX+DemoX+Demo_Course.tar.gz` instead of `/tmp/course.l8dtvuog.tar.gz`?

* This might require changes within the platform itself because the implementation doesn't support the configurable path generation. You can see the implementation [here](https://github.com/openedx/edx-platform/blob/open-release%2Fmaple.master/cms/djangoapps/contentstore/tasks.py#L338-L340). This change would require us to create our own tarball generation implementation though. (More than ~15 min)

OR 

2) If our end goal is to upload the files to s3 with Course ID as the name instead of the generated tarball's name. 
* We can customize this in the plugin and that should not be more than 15 mins. With this change the files uploaded on s3 will have course Id as name and the return URL in response would look like `"course-v1:MIT+MIT_IK_0001+R1": "https://defaultcourseexports.s3.amazonaws.com/course-v1:edX+DemoX+Demo_Course.tar.gz.tar.gz"` (Though we might have to omit some characters e.g. `-` that would not support a URL pattern)

Reply:
Thanks for the clarification, I don't care what the file name is on disk as long as we can customize how it gets represented once it lands in S3.

Reply:
This has been incorporated. now the uploaded files on s3 would have course_id as their name.